Source/SwiftLintFramework/Reporters/GroupedReporter.swift

```diff
@@ -0,0 +1,88 @@
+import SwiftyTextTable
+
+public struct GroupedReporter: Reporter {
+    // MARK: - Reporter Conformance
+
+    public static let identifier = "grouped"
```

Comment:
We should better make clear this is only useful as textual, human-readable output. Hence, something like `console-grouped` would fit better.

Reply:
Awesome!

Thank you for all the feedback! Really inspiring and good observations. I will get on these things as soon as I can.